libdnf.spec

```diff
@@ -13,7 +13,7 @@
 %global oldname libhif
 
 Name:           libdnf
-Version:        0.7.0
+Version:        0.7.1
```

Comment:
AFAIU we all agreed with bumping patch versions when there are two dependent PRs. It would make our nightly builds consistent and Neal can then release newer versions of libdnf more often because upstream would not be bounded so much to releasing in Fedora. We should just preserve ordering and officially make libdnf upstream 0.7.0 release. So the questions is rather *how do we tag releases in libdnf*? Currently it's done by git tag so there would be 2 places which would need to be modified when we bump version - spec and tag (with tito that would be easier).

So now we need to tag the current upstream `libdnf_0_7_0`, keep spec file changes in this PR, git tag `libdnf_0_7_1` after merge and everyone would be happy, right?